app/snippets/nrf52833-nosd/nrf52833-nosd.overlay

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/delete-node/ &sd_partition;
+/delete-node/ &code_partition;
+
+&flash0 {
+    partitions {
+        sd_partition: partition@0 {
+            reg = <0x0 DT_SIZE_K(4)>;
+        };
+        code_partition: partition@1000 {
+            reg = <DT_SIZE_K(4) DT_SIZE_K(428)>;
+        };
```

Comment:
I think it's worth adding a comment here about preserving the existing storage_partition with 0x6c000 offset and that accounting for the 428K size here.

Reply:
Added the comment about preserving the storage partition.